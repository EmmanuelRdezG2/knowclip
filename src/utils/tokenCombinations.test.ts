import { getTokenCombinations } from './tokenCombinations'

describe('getTokenCombinations', () => {
  it('gives all combinations for five items in right order', () => {
    expect(
      getTokenCombinations(['das', 'licht', 'spiegelt', 'sich', 'in'])
    ).toMatchSnapshot()
  })
})
