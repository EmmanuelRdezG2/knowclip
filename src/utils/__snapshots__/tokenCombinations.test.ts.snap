// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getTokenCombinations gives all combinations for five items in right order 1`] = `
Array [
  Array [
    "das",
    "licht",
    "spiegelt",
    "sich",
    "in",
  ],
  Array [
    "das",
    "licht",
    "spiegelt",
    "sich",
  ],
  Array [
    "das",
    "licht",
    "spiegelt",
    "in",
  ],
  Array [
    "das",
    "licht",
    "sich",
    "in",
  ],
  Array [
    "das",
    "spiegelt",
    "sich",
    "in",
  ],
  Array [
    "das",
    "licht",
    "spiegelt",
  ],
  Array [
    "das",
    "licht",
    "sich",
  ],
  Array [
    "das",
    "licht",
    "in",
  ],
  Array [
    "das",
    "spiegelt",
    "sich",
  ],
  Array [
    "das",
    "spiegelt",
    "in",
  ],
  Array [
    "das",
    "sich",
    "in",
  ],
  Array [
    "das",
    "licht",
  ],
  Array [
    "das",
    "spiegelt",
  ],
  Array [
    "das",
    "sich",
  ],
  Array [
    "das",
    "in",
  ],
]
`;
